json-rpc/src/tests/unit_tests.rs

```diff
@@ -315,6 +316,125 @@ proptest! {
         assert_eq!(event.sequence_number, 0);
         assert_eq!(event.transaction_version, 0);
     }
+    #[test]
+    fn test_get_transactions(blocks in arb_blocks_to_commit()) {
+        _test_get_transactions(blocks);
+    }
+
+    #[test]
+    fn test_get_account_transaction(blocks in arb_blocks_to_commit()) {
+        _test_get_account_transaction(blocks);
+    }
+}
+
+fn _test_get_transactions(blocks: Vec<(Vec<TransactionToCommit>, LedgerInfoWithSignatures)>) {
+    // set up MockLibraDB
+    let mock_db = mock_db(blocks);
+
+    // set up JSON RPC
+    let address = format!("0.0.0.0:{}", utils::get_available_port());
+    let _runtime = bootstrap(
+        address.parse().unwrap(),
+        Arc::new(mock_db.clone()),
+        channel(1).0,
+    );
+    let client = reqwest::blocking::Client::new();
+    let url = format!("http://{}", address);
+
+    let version = mock_db.get_latest_version().unwrap();
+    let page = 800usize;
+
+    for base_version in (0..version).map(u64::from).take(page).collect::<Vec<_>>() {
+        let request = serde_json::json!({"jsonrpc": "2.0", "method": "get_transactions", "params": [base_version, page as u64, false], "id": 1});
+        let resp = client.post(&url).json(&request).send().unwrap();
+        assert_eq!(resp.status(), 200);
+        let resp_json: JsonMap = resp.json().unwrap();
```

Comment:
nit: I think we can combine these two lines using `fetch_data`

Reply:
fetch_data can't be used, because in this case it is actually an Value::Array() , but i got rid of unnecessary variable now